### DISCUSSION TITLE
Is there any way to export api types?

### DISCUSSION THREAD

**Discussion by wxf1024** [honojs/hono#2400] [Q&A] [answered]
For example, I have a api like:
``` js
app.get('/user', c => {
  c.json({
    name: 'ppz',
    year: 3,
  })
})
```
Is there any way to export api types? So I can use it in the client code.

**Comment by yusukebe** [comment]
Hi @wxf1024 

See: https://hono.dev/guides/rpc

**Reply by wxf1024** [reply]
thank you very much!

but sorry for not describing my question exactly.  
I mean, any way that can run in the browser?

**Reply by yusukebe** [reply, marked as answer]
Yes! If you set up it properly, you could.

**Reply by wxf1024** [reply]
wow, I'll try it, thx!
